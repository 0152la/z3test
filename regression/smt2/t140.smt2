(set-option :auto-config true)
(set-option :produce-models true)
(declare-const a Int)
(assert (> a 2))
(check-sat)
(get-model)
