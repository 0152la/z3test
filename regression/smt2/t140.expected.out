sat
(model 
  (define-fun a () Int
    3)
)
